wordpress-mobile/WordPress-iOS
# Comment Moderation Bar: add Approved, Spam, and Trash buttons.

Ref: #17200 
Depends on: #17211

This adds the Approved, Spam, and Trash buttons to the moderation bar. Tapping the buttons will toggle the UI to indicate selected or not. No action is actually performed yet.

NOTE: The vertical dividers can show incorrectly when two buttons are selected. This isn't a real scenario since a comment can have only one state. It will be resolved once the comment state is applied.

To test:
- Enable the `newCommentDetail` feature.
- Go to My Site > Comments > Comment details.
- Verify all the buttons are in the unselected state.
- Tap a button. 
  - Verify the UI changes to the selected state.
  - Verify the vertical dividers on either side are hidden when selected, shown when unselected.

| Unselected | Selected |
|--------|-------|
| <img width="441" alt="light_unselected" src="https://user-images.githubusercontent.com/1816888/134593313-4be3e8bf-4449-4642-b2e1-fb13ad9fc947.png"> | <img width="437" alt="light_selected" src="https://user-images.githubusercontent.com/1816888/134593333-0abc0056-ddc3-4a99-8807-0ebff9fcc60a.png"> |
| <img width="447" alt="dark_unselected" src="https://user-images.githubusercontent.com/1816888/134593460-f0706d93-d9f3-4f1b-b758-0b4a06b61f00.png"> | <img width="452" alt="dark_selected" src="https://user-images.githubusercontent.com/1816888/134593578-4b99095d-74d0-4dc8-a6ad-b2be3e9e517b.png"> |

## Regression Notes
1. Potential unintended areas of impact
N/A. Feature is incomplete and disabled.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. Feature is incomplete and disabled.

3. What automated tests I added (or what prevented me from doing so)
N/A. Feature is incomplete and disabled.

PR submission checklist:
